### PR TITLE
[T5A1][T10-B3]ZavierOng

### PR DESCRIPTION
**Pull request by E0002878**

### CODE REVIEW

**Comment by E0002878** [comment]
Ready for review

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -73,13 +78,36 @@ Use case ends.
...
+  Use case resumes at step 2
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should have an user friendly interface that is not confusing.
+6. Should not exceed more than 5 seconds between commands.
```
Comment (mauris) [inline doc/DeveloperGuide.md]: good to be specific

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -73,13 +78,36 @@ Use case ends.
...
+> 3a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
 
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should have an user friendly interface that is not confusing.
```
Comment (mauris) [inline doc/DeveloperGuide.md]: as mentioned in class, a "user friendly interface" is very subjective.

**Comment by mauris** [comment]
Some comments added. Feel free to comment if you have any questions. Once done, you may the PR after reading comments.

**Pull request closed without merging**